index.html

```diff
@@ -294,6 +294,7 @@
     <p>Source available on <a href="https://github.com/jrconlin/WebPushDataTestPage">GitHub</a>. Licensed under <a href="https://www.mozilla.org/en-US/MPL/2.0/">Mozilla Public License v.2.0</a></p>
 </div>
   </body>
+  <script src="common.js"></script>
   <script src="base64.js"></script>
   <script src="webpush.js"></script>
   <script src="vapid.js"></script>
```

Comment:
Starting to feel like we should make modules out of these? Could be follow-up issue.